src/site/content/en/learn/html/lists/index.md

```diff
@@ -3,10 +3,290 @@ title: 'Lists'
 authors:
   - estelleweyl
 description: Lists and other ways of grouping your content.
-date: 2022-09-08
+date: 2022-12-02
 placeholder: true
 tags:
   - html
 ---
 
-Coming soon!
\ No newline at end of file
+Lists are more common than you might think. If you've ever taken a programming class, the first project may have been to create a
+shopping list or a to-do list. Those are lists. Multiple-choice tests are generally numbered lists of questions: the multiple possible answers
+for each question are nested lists.
+
+HTML provides us with a few different ways to mark up lists. There are ordered lists (`<ol>`), unordered lists (`<ul>`), and description lists (`<dl>`).
+List items (`<li>`) are nested within ordered lists and unordered lists. Inside a description list, you'll find description terms (`<dt>`) and description
+details `<dd>.` We'll cover all of these here.
+
+In HTML forms, lists of `<option>` elements make up the content of `<datalist>`, `<select>`, and `<optgroup>` within a `<select>`. These are discussed in [HTML forms](/learn/forms).
+
+In menus and unordered lists, list items are usually displayed using bullet points. In ordered lists, they are usually preceded
+by an ascending counter such as a number or letter. The bullets and numbering order can be controlled or reversed with HTML or CSS,
+or both.
+
+By default, ordered and unordered list items are prefixed with numbers or bullets. But even when you don't want lists to look like lists,
+you still want a list of items, like in navigation bars, a to-do list with checkboxes instead of bullets, or true and false questions
+in a multiple-choice test. For all of these lists without bullets, it is appropriate to use HTML list elements.
+
+## Unordered lists
+
+The `<ul>` element is the parent element for unordered lists of items. The only children of a `<ul>` are one or more `<li>` list
+item elements. Let's create a list of machines. We use an unordered list because the order doesn't matter (don't tell them that):
+
+```html
+<ul>
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ul>
+```
+<img src="image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/OeZKatnZZPbs7q0yuM2C.png" width="196" height="146" alt="Blender, toaster, vacuum">
+
+By default, each unordered list item is prefixed with a bullet. The unordered list has no element-specific attributes.
+You need to close out your lists with a `</ul>`.
+
+## Ordered lists
+
+The `<ol>` element is the parent element for ordered lists of items. The only children of an `<ol>` are one or more `<li>` elements, or list items.
+The "bullets" in this case, though, are numbers of a multitude of types. The type can be defined in CSS with the `list-style-type` property
+or via the `type` attribute.
+
+```html
+<ol>
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ol> https://codepen.io/estelle/pen/OJEXLWK
+
+<ol type="A">
+   <li>Blender</li>
+   <li>Toaster</li>
+   <li>Vacuum</li>
+</ol> https://codepen.io/estelle/pen/OJEXLWK
```

Comment:
Are these codepen links meant to be here?

Reply:
i added the codepen links to show the resource if y'all wanted to include a codepen; they should not be displayed as text though.

Reply:
@rachelandrew Let me know what you think about including these codepens! 

Thanks for the comments @estelle I'll make sure to get this stuff updated on Monday.